### DISCUSSION TITLE
Redux Essentials Part 8

### DISCUSSION THREAD

**Discussion by hasanradi93** [mswjs/msw#1279] [Q&A] [answered]
### Scope

Improves an existing behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

Can you set please on GitHub the Result for the Part 8 
Because I have error with UseSice.js page in the endpoint section, and I can't resolve it.

**Comment by kettanaito** [comment, marked as answer]
Hey, @hasanradi93. 

It seems you are referring to some sort of eduction material that includes MSW but is not written by its maintainers. I'm sorry to hear you have struggles with resolve it but I highly encourage you to reach out to the authors of that material and ask them for help, if they provide any. 

I can address issues with the library but I can't solve tasks for you. Thank you for understanding.
